### ISSUE TITLE
Cannot read properties of undefined (reading 'viewType')

### ISSUE THREAD

**Issue by Glecun** [McSodbrenner/vscode-better-open-editors#20]
Hi ! I installed the extension and I have this error when I try tu use it.
![image](https://github.com/McSodbrenner/vscode-better-open-editors/assets/7093472/c029ccf9-bff9-42ec-aeaf-1800648fee35)

**Comment by McSodbrenner** [comment]
Interesting. Do you notice any problems when using the plugin? Does it only occur with certain files?

**Comment by analogue** [comment]
Attaching error log:
```
2023-07-27 12:47:43.993 [info] ExtensionService#_doActivateExtension McSodbrenner.better-open-editors, startup: false, activationEvent: 'onStartupFinished'
2023-07-27 12:47:44.007 [error] Activating extension McSodbrenner.better-open-editors failed due to an error:
2023-07-27 12:47:44.007 [error] TypeError: Cannot read properties of undefined (reading 'viewType')
	at /Users/semirpatel/.vscode/extensions/mcsodbrenner.better-open-editors-1.10.1/src/treeviewPanel.js:112:18
	at Array.filter (<anonymous>)
	at TreeviewPanel.recreateTree (/Users/semirpatel/.vscode/extensions/mcsodbrenner.better-open-editors-1.10.1/src/treeviewPanel.js:110:15)
	at new TreeviewPanel (/Users/semirpatel/.vscode/extensions/mcsodbrenner.better-open-editors-1.10.1/src/treeviewPanel.js:25:8)
	at activate (/Users/semirpatel/.vscode/extensions/mcsodbrenner.better-open-editors-1.10.1/src/extension.js:6:24)
	at Tn.eb (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:113:13743)
	at Tn.db (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:113:13456)
	at /Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:113:11577
	at async w.n (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:99:6194)
	at async w.m (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:99:6157)
	at async w.l (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:99:5614)
2023-07-27 12:48:00.847 [error] TypeError: Cannot read properties of undefined (reading 'update')
	at /Users/semirpatel/.vscode/extensions/mcsodbrenner.better-open-editors-1.10.1/src/treeviewPanel.js:67:15
	at Array.forEach (<anonymous>)
	at i.value (/Users/semirpatel/.vscode/extensions/mcsodbrenner.better-open-editors-1.10.1/src/treeviewPanel.js:61:18)
	at p.w (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:60:1902)
	at p.x (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:60:1972)
	at p.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:60:2188)
	at e.$7$b.$acceptTabOperation (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:101:18302)
	at t.N (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:11199)
	at t.M (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:10965)
	at t.H (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:10046)
	at t.G (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:9122)
	at i.value (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:111:7952)
	at p.w (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:60:1902)
	at p.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:60:2119)
	at f.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:76:13931)
	at i.value (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:127:30341)
	at p.w (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:60:1902)
	at p.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:60:2119)
	at f.fire (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:76:13931)
	at MessagePortMain.<anonymous> (/Applications/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:127:28621)
	at MessagePortMain.emit (node:events:513:28)
	at MessagePortMain._internalPort.emit (node:electron/js2c/utility_init:2:367)
```

**Comment by no-identd** [comment]
@McSodbrenner please could you code review & test @OrkWard's #21, merge it, and publish a new release? It's very annoying and makes the extension essentially useless. I was very excited when I just found it but found it doesn't work at all for me, almost certainly due to this issue :(

**Comment by McSodbrenner** [comment]
Fixed in 1.10.2
